readme.md

```diff
@@ -70,7 +70,7 @@ There are currently three additional options that can be passed to `b.dependency
 For example, here is a `b.dependency()` call that and links against a shared Lua 5.2 library:
 
 ```zig
-const ziglua = b.dependency("ziglua", .{
+const ziglua = b.dependency("lua_wrapper", .{
```

Comment:
I guess you should change ziglua here to lua_dep too.

Reply:
oooh, good catch, thanks!